### ISSUE TITLE
Code of Conduct Transparency Report

### ISSUE THREAD

**Issue by gdequeiroz** [RConsortium/RCDI-WG#11]
I saw this today and found it very relevant: https://pycon.blogspot.com/2018/06/pycon-2018-code-of-conduct-transparency.html

**Comment by stefaniebutland** [comment]
Indeed. The rOpenSci Code of Conduct Committee has agreed to publish an annual transparency report modeled on that one. We also plan to publish brief, anonymized summaries of incidents, after they are resolved, on our forum in a dedicated read-only category (while 100% upholding people's privacy).

**Comment by sdruskat** [comment]
Something of a similar nature has been published for DjangoCon 2018: https://2018.djangocon.eu/news/coc-transparency/.
